<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDomain</class>
 <widget class="QMainWindow" name="frmDomain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>观察域</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxDomain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>求解域选择</string>
    </property>
    <widget class="QCheckBox" name="chk1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>电磁场</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>时间步数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTimeStepNum">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>热/结构</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>时间步数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTimeStepNum2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtTimeStepNum</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
